File: 2015_11_16_Social_Tunes_Sharer/android/MyApplication/app/src/main/java/io/intrepid/socialtunes/MainActivity.java
package io.intrepid.socialtunes;

import android.graphics.drawable.Drawable;
import android.os.Bundle;
import android.support.v7.app.AppCompatActivity;

public class MainActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

        Drawable drawable = getResources().getDrawable(R.drawable.big_pile_of_leaves_blur, getTheme());
        if (drawable != null) {
            drawable.setAlpha(160);
            getWindow().setBackgroundDrawable(drawable);
        }
    }
}

File: 2015_11_16_Social_Tunes_Sharer/android/MyApplication/app/src/main/java/io/intrepid/socialtunes/MainActivity.kt
package io.intrepid.socialtunes

import android.os.Bundle
import android.support.v7.app.AppCompatActivity

class MainActivity : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)

        setContentView(R.layout.activity_main)

        val drawable = resources.getDrawable(R.drawable.big_pile_of_leaves_blur, theme)
        if (drawable != null) {
            drawable.alpha = 160
            window.setBackgroundDrawable(drawable)
        }
    }
}
